docs/testing/README.md

```diff
@@ -0,0 +1,26 @@
+# Testing in LocalStack
+
+- [Integration Tests](integration-tests/README.md)
+- [Parity Testing](parity-testing/README.md)
+- [Multi-account and Multi-region Testing](multi-account-region-testing/README.md)
+- [Terraform Tests](terraform-tests/README.md)
+
+## Rules for stable tests
+
+Through experience, we encountered some guiding principles and rules when it comes to testing LocalStack.
+These aim to ensure a stable pipeline and keep flakes minimal. Any newly added test and feature should keep these in mind!
+
+| **ID** | **Rule** |
+|-------|-----------|
+| R01   | Inform code owners and/or test authors when a test is marked as flaky so that it can be fixed ASAP. |
+| R02   | Do not assume external dependencies (AWS resources, files, packages, images, licenses) are indefinitely available. |
```

Comment:
this one is not clear to me. could you elaborate/rephrase? why would i assume indefinitely available resources in a test? is there an example?

Reply:
We have tests which access remote resources. Some access our own resources, like tests surrounding pods, and some access some files (like libraries, container images) stored on servers which are not under our control.